[2021-10-05 14:04:36,187] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_A 2021-10-05T14:04:35.342710+00:00 [queued]>
[2021-10-05 14:04:36,226] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_A 2021-10-05T14:04:35.342710+00:00 [queued]>
[2021-10-05 14:04:36,228] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-05 14:04:36,230] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-05 14:04:36,232] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-05 14:04:36,276] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_A> on 2021-10-05T14:04:35.342710+00:00
[2021-10-05 14:04:36,326] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_A', '2021-10-05T14:04:35.342710+00:00', '--job-id', '138', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpse2i0end', '--error-file', '/tmp/tmp0nrzxanm']
[2021-10-05 14:04:36,330] {standard_task_runner.py:77} INFO - Job 138: Subtask Training_Model_A
[2021-10-05 14:04:36,291] {standard_task_runner.py:52} INFO - Started process 82089 to run task
[2021-10-05 14:04:36,586] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_A 2021-10-05T14:04:35.342710+00:00 [running]> on host d6b4131456ad
[2021-10-05 14:04:36,751] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-10-05T14:04:35.342710+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-05T14:04:35.342710+00:00
[2021-10-05 14:04:36,767] {python.py:151} INFO - Done. Returned value was: None
[2021-10-05 14:04:36,791] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_A, execution_date=20211005T140435, start_date=20211005T140436, end_date=20211005T140436
[2021-10-05 14:04:36,851] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-05 14:04:36,922] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
